### DISCUSSION TITLE
Browser integration doesn't work anymore since ubuntu 22.04 update

### DISCUSSION THREAD

**Discussion by jeanmartin13** [keepassxreboot/keepassxc#8536] [Q&A] [answered]
Hello,

Since I updated from ubuntu 20.04 to 22.04 the browser integration in firefox doesn't work anymore. Can't load the database.

**Comment by droidmonkey** [comment, marked as answer]
You can't use a snap installed Firefox... yet. They need to release their new snap that enables native messaging.

**Reply by jeanmartin13** [reply]
Thanks, that is what I thought. Is there a firefox version I can us on 22.04 or do I just have to wait for an update ?

**Reply by droidmonkey** [reply]
Use the natively installed version, you might need to use a ppa for that.
